salt/modules/boto_iam.py

```diff
@@ -46,6 +46,7 @@
 # Import salt libs
 import salt.utils.compat
 import salt.utils.odict as odict
+import salt.utils.boto
```

Comment:
Sigh. I really wish PRs would stop getting merged with this. It's not backwards compatible and it doesn't make it possible to override the modules in core. It's recommended to use __utils__['boto.function'] for accessing utils.